src/Components/Browser.JS/Microsoft.AspNetCore.Components.Browser.JS.npmproj

```diff
@@ -8,14 +8,24 @@
 
   <ItemGroup>
     <WebpackInputs Include="**\*.ts" Exclude="node_modules\**" />
+    <WebPackOutputs Include="src\dist\components.webassembly.js" />
+    <WebPackOutputs Include="src\dist\components.server.js" />
```

Comment:
Nit: for consistency, `WebpackOutputs `

Reply:
I'll do it in a followup. I'd rather not wait another day to fix up casing 😄